issue_template/feature_request.yml

```diff
@@ -0,0 +1,45 @@
+name: 🛠️ Feature Request
+description: Suggest an idea to improve something
+title: "[Feature]: "
```

Comment:
Following the other 2 templates, shouldn't this be `[Feature]: <title>` instead?